src/resources/parser/texture/img.js

```diff
@@ -23,7 +26,9 @@ class ImgParser {
         // - Firefox doesn't support options parameter to createImageBitmap (see https://bugzilla.mozilla.org/show_bug.cgi?id=1533680)
         // - Safari supports ImageBitmap only as experimental feature.
         // this.useImageBitmap = typeof ImageBitmap !== 'undefined' && /Firefox/.test(navigator.userAgent) === false;
-        this.useImageBitmap = false;
+        // WebGPUTexture expects ImageBitmap.
+        const isWebGPU = registry?._loader?._app?.graphicsDevice?.deviceType === DEVICETYPE_WEBGPU;
```

Comment:
might be better to pass device type to handlers?

Reply:
this is temporary I think, as you wanted to look at enabling this for webgl too?